examples_ch59x/blink/funconfig.h

```diff
@@ -1,11 +1,14 @@
 #ifndef _FUNCONFIG_H
 #define _FUNCONFIG_H
 
-#define FUNCONF_USE_HSI         0 // CH592 does not have HSI
-#define FUNCONF_USE_HSE         1
-#define FUNCONF_DEBUG_HARDFAULT 0
-#define FUNCONF_USE_CLK_SEC     0
-#define FUNCONF_USE_DEBUGPRINTF 0 // saves 16 bytes, enable / remove if you want printf over swio
-#define FUNCONF_INIT_ANALOG     0 // ADC is not implemented yet
+#define FUNCONF_USE_HSI           0 // CH592 does not have HSI
+#define FUNCONF_USE_HSE           1
+#define CLK_SOURCE_CH59X          CLK_SOURCE_PLL_60MHz // default so not really needed
+#define FUNCONF_SYSTEM_CORE_CLOCK 60 * 1000 * 1000     // keep in line with CLK_SOURCE_CH59X
```

Comment:
I would, here, comment any of the thing that default and just say // defaults are commented out.

It's very helpful to have this reference though!

Reply:
After this work I am slightly inclined to have the funconfig.h in the examples have more references like this actually, but it's totally your call of course.

Reply:
> After this work I am slightly inclined to have the funconfig.h in the examples have more references like this actually (I much more often read code + comments than documentation), but it's totally your call of course.